config/application.config.php

```diff
@@ -3,6 +3,7 @@
     // This should be an array of module namespaces used in the application.
     'modules' => [
         'ZendDeveloperTools',
+        'TestData',
```

Comment:
#410 adds something which only loads development related modules in development environments, so this won't be a problem.